### PR TITLE
Document nginx-ingress auth-url snippet

### PR DESCRIPTION
**Pull request by vic3lord**
Fixes https://github.com/linkerd/linkerd2/issues/3619

### CODE REVIEW

File: linkerd.io/content/2/tasks/using-ingress.md [hunk 1]
```diff
@@ -71,6 +71,11 @@ The important annotation here is:
     nginx.ingress.kubernetes.io/configuration-snippet: |
       proxy_set_header l5d-dst-override $service_name.$namespace.svc.cluster.local:$service_port;
       grpc_set_header l5d-dst-override $service_name.$namespace.svc.cluster.local:$service_port;
+
+    # NOTE: if you are using auth-url you'd need to add this snippet as well.
```
Comment (cpretzer) [inline linkerd.io/content/2/tasks/using-ingress.md]: @vic3lord Thanks for submitting this PR. Would you mind linking to the `auth-url` docs in order to provide more context?

Comment (vic3lord) [reply linkerd.io/content/2/tasks/using-ingress.md]: Of course, where would I put it though? Inside the yaml block?

**Comment by cpretzer** [comment]
I’d just do it as a link in the markdown for the “auth-url” text

**Comment by vic3lord** [comment]
Signed DCO

File: linkerd.io/content/2/tasks/using-ingress.md [hunk 2]
```diff
@@ -73,6 +73,17 @@ The important annotation here is:
       grpc_set_header l5d-dst-override $service_name.$namespace.svc.cluster.local:$service_port;
 ```
 
+{{< note >}}
+If you are using [auth-url](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#external-authentication)
+you'd need to add the following snippet as well.
+{{< /note >}}
+
+```yaml
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/using-ingress.md]: Mind moving this into the note?

Comment (vic3lord) [reply linkerd.io/content/2/tasks/using-ingress.md]: As yaml block or just text?

Comment (grampelberg) [reply linkerd.io/content/2/tasks/using-ingress.md]: As the YAML block, just so that it is visually grouped.

Comment (vic3lord) [reply linkerd.io/content/2/tasks/using-ingress.md]: Done

**Comment by grampelberg** [comment]
Looks like lint wants you to add a newline between the backticks and the end of the note.

**Comment by grampelberg** [comment]
Thanks!

**Pull request merged**